src/bend.ts

```diff
@@ -270,7 +271,7 @@ export class Bend extends Modifier {
           renderRelease(start.x, start.y, last_drawn_width, bend_height);
         }
 
-        if (last_bend === null) {
+        if (!last_bend) {
```

Comment:
@rvilarl @0xfe I think this is fine for now, since it fixes the bug at least.

However, this line and lines above that start with

`if (last_bend`

almost make me think last_bend is a boolean type. Maybe we should actually have isUndefined() and isNotUndefined() utility functions??? 😬

Reply:
@ronyeh I answered in 
https://github.com/0xfe/vexflow/issues/1003#issuecomment-876529023
what do you think?